experiments/benchmarks/richards.jl

```diff
@@ -0,0 +1,372 @@
+# # Global run of RichardsModel
```

Comment:
Could we add this?

"In this simulation, we have turned lateral flow off because horizontal boundary conditions and the land/sea mask are not yet supported by ClimaCore."

Reply:
Maybe could we also add the pertinient info re: timestepping and discretization? (n elements, n layers, dt, timestepper, max iters,?)

Reply:
maybe even the tolerance and how often we update the Jacobian